install.sh

```diff
@@ -10,39 +11,90 @@ elif [ $PLATFORM == "Linux" ]; then
 	PLATFORM="ubuntu-20_04-x86-64"
 else
 	echo "Scripted installation on your platform is not supported."
-	echo "See compiled binaries in the latest release: https://github.com/simplex-chat/simplex-chat/releases/latest"
+	echo "See compiled binaries in the latest release: https://github.com/$APP_NAME/$APP_NAME/releases/latest"
 	exit 1
 fi
 
-[ ! -d $TARGET_DIR ] && mkdir -p $TARGET_DIR
+# / Prepare to upgrade from v0 to v1
+
+# Determine path of chat binary
+if [[ -n "$(which $APP_NAME)" ]]; then
+	binary=$(which $APP_NAME)
+elif [[ -f "$BIN_PATH" ]]; then
+	binary=$BIN_PATH
+else
+	binary=""
+fi
+
+# If chat binary not found, check v0 initial migration and offer to abort or continue
+if [[ -z $binary ]]; then
+	agent_db="$HOME/.simplex/simplex.agent.db"
+	if [[ \
+		-f "$agent_db" && \
+		$(echo "select * from migrations;" | sqlite3 $agent_db | grep 20210101_initial) \
+	]]; then
+		echo "Found initial migration of the previous version of SimpleX Chat, current version v1.0.0 is not compatible."
+		echo "If you choose to continue the installation current version will be installed as $APP_NAME with clean database, old database will be preserved."
```

Comment:
```suggestion
		echo "If you continue, the current version will be installed as $APP_NAME with a clean database, the old database will be preserved."
```

"if you choose to accept this mission"...